src/components/Modal.js

```diff
@@ -1,16 +1,15 @@
 import { useEffect, useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
-//
 import { ReactComponent as CloseIcon } from '../icons/close.svg';
 import { ReactComponent as Help } from '../icons/helpcenter-icon.svg';
 import HelpCenter from './HelpCenter';
+import Footer from './Footer';
 
 import { toggleHelp } from '..';
 import { CapabilityAPI, LocalStorageUtils } from '../utils';
 
 const Modal = ( { onClose } ) => {
-	const [ brand, setBrand ] = useState( '' );
-	const [ refresh, setRefresh ] = useState( false );
+	const [ , setBrand ] = useState( '' );
```

Comment:
Can this state brand removed? It is not used in this component and not shared with other components.